### ISSUE TITLE
Virtual Keyboard (Applet Overlays)

### ISSUE THREAD

**Issue by BurAndBY** [Ryujinx/Ryujinx#2877]
#### Overview:
Applet overlays, such as keyboards, error messages (like RYU ones)

#### Smaller Details: 
- If applet like Mii exists, I think this is possible.

#### Nature of Request:  
- Change
Replace keyboard window with keyboard applet
![image](https://user-images.githubusercontent.com/48630651/143221864-bff1724f-ff3c-4574-b4bc-333639120e18.png)

Replace RYU-xxxx errors to NX-styled ones
![image](https://user-images.githubusercontent.com/48630651/143222132-868b8c5f-69d7-4dcb-b259-9f2dd6580a79.png)

### Why would this feature be useful?
It makes big emulation easier. It feels bad when you play and to enter something you need to get to your keyboard and write it there.

**Comment by RandomDude560** [comment]
Yes having to get up and use the keyboard just to name your horse in BOTW is kind of annoying.
